src/Core/Geometry/TriangleMesh.cpp

```diff
@@ -36,6 +40,11 @@ void AttribArrayGeometry::colorize( const Utils::Color& color ) {
     getAttrib( colorAttribHandle ).setData( Vector4Array( vertices().size(), color ) );
 }
 
+void AttribArrayGeometry::invalidateAabb() {
```

Comment:
Why is this method not declared virtual in `AbstractGeometry` that has the `computeAabb` and `getAabbObservable`method declaration ?